src/pages/BookDetails/BookDetails.module.scss

```diff
@@ -0,0 +1,7 @@
+.gridContainer {
+  background-color: lightgrey;
```

Comment:
let's add a `styles.scss` file in the `src/` directory with some global styles, like global an app color palette. 
https://sass-lang.com/documentation/variables
then in this file you add 
```
@import 'src/styles';
```
and use the variables